### PR TITLE
Another change

### PR DESCRIPTION
**Pull request by HackerGal**
Added a few more lines of code to practice pull and push requests using VS Code.

### CODE REVIEW

File: python/helloworld.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+
+for i in range(2,101):
+    x=1
+    for j in range(2, i):
+        n=i%j
+        if n==0:
+            x=0
+            break
+    if x==1:
+        print (i)
```
Comment (chadwpetersen) [inline python/helloworld.py]: Add an empty line here.

**Review by chadwpetersen — CHANGES_REQUESTED** [review]
It is always great to add detail to the pull request description as to what the changes in the code means. What is the code doing -so that I can review it based on that context.

**Comment by chadwpetersen** [comment]
Also it is always great to create functions so that your code is housed in a file and a function that makes sense.

File: python/helloworld.py [hunk 2]
```diff
@@ -1,3 +1,18 @@
 # Short code written in python
+# Code added to print a prime number on each line
 
-print("My name is Melanie")
+a= int(input("Insert any number more than 2: "))
```
Comment (chadwpetersen) [inline python/helloworld.py]: I suggest we could have just created another file -called `primelist.py` in this folder and have this code move there and leaving the initial print code in this file. Such that you have two files each serving two different purposes.

**Review by chadwpetersen — CHANGES_REQUESTED** [review]

File: python/helloworld.py (hunk 2 above)
Comment (HackerGal) [reply python/helloworld.py]: Separate file made for the primeList function. I hope this is adequate.

Comment (chadwpetersen) [reply python/helloworld.py]: The helloworld file should just print out hello world and the other file should have all of this new code in it.

**Review by chadwpetersen — CHANGES_REQUESTED** [review]

**Review by chadwpetersen — APPROVED** [review]

**Pull request merged**